### PR TITLE
Add Descriptive Section to CC Search About Page

### LINKED ISSUE #175: [Feature] Create a CC Search About Page
## Problem
Current CC Search doesn't have a descriptive section or page

## Description
Create a descriptive page for CC Search

This is a main issue that links to a sub-issue:
#73 

## Implementation

- [ ] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by SylviaUzoh1**
## Fixes

- Fixes #175  by @PreciousOritsedere 

## Description

This pull request adds a descriptive section to CC Search, addressing the lack of clear information about the platform and its functionality. The new section provides users with a concise overview of CC Search, explaining its purpose and how it helps users find openly licensed content. This enhancement improves the overall user experience by offering essential information directly on the page.

## Technical details

- Followed a minimal design pattern to keep the page clean and simple, ensuring that the search page remains user-friendly and not overcrowded with text.
- Focused on providing essential information with concise text, ensuring that users can quickly understand the tool's purpose without being overwhelmed.
- Created a new component for the descriptive section about CC Search.
- Implemented the new section inside the main tag for optimal layout and accessibility while considering mobile responsiveness.
- Styled the section using a dedicated class in `style.css` to maintain consistent design with the rest of the platform.
- Ensured visual consistency by aligning the new section with the overall design and aesthetic of CC Search.

## Tests

- Ran the application on a local server to ensure that the new section displays correctly and functions as intended across various devices and screen resolutions.
- Verified that all layout elements render properly in modern browsers.
- Checked the responsive behavior to confirm usability on mobile, tablet, and desktop views.

## Screenshots

CC About Page Mobile
CC About Desktop

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -158,6 +158,15 @@ <h3>Find content you can share, use, and remix.</h3>
               placeholder="Enter your search query" />
             <button class="search-button">Search</button>
           </div>
+          <div class="body-search-description" role="contentinfo">
+            <p>
```
Comment (possumbilities) [inline src/index.html:162]: Let's reword this to: 

The Search Portal helps you search CC licensed works across multiple platforms.

Comment (possumbilities) [reply src/index.html:162]: Let's also remove the "learn more" sentence at the end for now.

**Comment by possumbilities** [comment]
@SylviaUzoh1 Great work here, there's just some minor copy changes and I think this should be closer to approval!.

**Comment by SylviaUzoh1** [comment]
> @SylviaUzoh1 Great work here, there's just some minor copy changes and I think this should be closer to approval!.

@possumbilities, thank you for the guide. I have updated the pull request with the text as requested.

**Review by possumbilities — APPROVED** [review]
Wonderful! Approved.

**Comment by possumbilities** [comment]
@SylviaUzoh1 if you comment on the original Issue I can assign you!
